packages/icons-react/src/IconFactory.tsx

```diff
@@ -20,8 +21,10 @@ export const makeIconComponent = (variants: IconVariants) => {
         const MediumBoldIconComponent = variants["bold"]["medium"];
         const SmallIconComponent = variants["regular"]["small"];
         const SmallBoldIconComponent = variants["bold"]["small"];
+
+        const Tag = as;
```

Comment:
Personlig så blander jeg `Tag` med `@fremtind/jkl-tag`. Kan `El` være et alternativ?

Reply:
Jeg har sterke meninger om mye rart, men ikke akkurat dette.

Men jeg kalte den `Tag` da jeg gjorde det samme for `ExpandButton`

Reply:
Ja, det gir mening. Men når jeg leste markupen så ble jeg i stuss 😅

Reply:
Koster meg veldig lite å rename den men da må jeg gjøre det i `ExpandButton` også syns jeg, så det er litt konsekvent.

Reply:
Enig

Reply:
Dånn